Casks/f/fossa.rb

```diff
@@ -0,0 +1,18 @@
+cask "fossa" do
+  version "3.8.18"
+  sha256 "8fba90188752cea2c35a764f3b2308adc71058d7984cd761dd447a1c8359444f"
+
+  url "https://github.com/fossas/fossa-cli/releases/download/v#{version}/fossa_#{version}_darwin_amd64.zip",
+      verified: "github.com/fossas/fossa-cli/"
+  name "fossa"
```

Comment:
@meghfossa Should this be stylised as below?

```suggestion
  name "FOSSA"
```

Reply:
I would prefer it to be `fossa` for now (same as our previous custom tap).